morse_code.rb

```diff
@@ -0,0 +1,30 @@
+MORSE_CODE = { 'A' => '.-', 'B' => '-...', 'C' => '-.-.', 'D' => '-..', 'E' => '.', 'F' => '..-.',
+               'G' => '--.', 'H' => '....', 'I' => '..', 'J' => '.---', 'K' => '-.-', 'L' => '.-..',
+               'M' => '--', 'N' => '-.', 'O' => '---', 'P' => '.--.', 'Q' => '--.-', 'R' => '.-.',
+               'S' => '...', 'T' => '-', 'U' => '..-', 'V' => '...-', 'W' => '.--', 'X' => '-..-',
+               'Y' => '-.--', 'Z' => '--..', '0' => '-----', '1' => '.----', '2' => '..---',
+               '3' => '...--', '4' => '....-', '5' => '.....', '6' => '-....', '7' => '--...',
+               '8' => '---..', '9' => '----.' }.freeze
+
+def decode_char(char)
+  MORSE_CODE.key(char)
+end
+
+def decode_word(word)
+  result = ''
+  characters = word.split
+  characters.each { |char| result += char.split.map { |code| decode_char(code) }.join }
+  result
+end
```

Comment:
- [ ] The expected result for the word is different from your result, kindly fix it.

Required Output
![image](https://user-images.githubusercontent.com/81932478/214824484-77354426-203e-4d7f-86e4-99fd29181ab1.png)

Your Output
![image](https://user-images.githubusercontent.com/81932478/214824563-648fffd7-c415-4e94-be96-edb8ab33ecb7.png)